prisma/schema.prisma

```diff
@@ -83,3 +84,28 @@ model OpenAIModels {
     User                        User?     @relation(fields: [userId], references: [id])
     userId                      String?
 }
+
+model ImageGenerationRequest {
+    id              String            @id @default(cuid())
+    createdAt       DateTime          @default(now())
+    requestText     String
+    numberOfImages  Int
+    generatedImages GeneratedImages[]
+    User            User?             @relation(fields: [userId], references: [id])
+    userId          String?
+    ImageSize       ImageSize?        @relation(fields: [imageSizeTitle], references: [size])
+    imageSizeTitle  String?
+}
+
+model GeneratedImages {
+    id                       String                  @id @default(cuid())
+    createdAt                DateTime                @default(now())
+    imageUrl                 String
+    ImageGenerationRequest   ImageGenerationRequest? @relation(fields: [imageGenerationRequestId], references: [id])
+    imageGenerationRequestId String?
+}
+
+model ImageSize {
+    size                   String                   @id
```

Comment:
Add column size_id and use that as the relationship

Reply:
Removed the image size table, use string for image sizes in the requests table